frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -144,6 +151,52 @@ function positionToString(pos: Vector3): string {
   return pos.map(value => roundTo(value, 2)).join(", ");
 }
 
+function shortcutBuilder(shortcuts: Array<string>): Node {
+  const lineColor = "var(--ant-text-primary)";
```

Comment:
(hm, I typed that already somewhere but my comment seems to have been lost)

can you adapt this color to be more subtle/secondary? then, the shortcuts are less emphasized (which is good since the actual item should be primary) and it also fits the mouse icons which are a bit more greyish.